#1. Produce the error. 
#2. $error[0].exception.GetType().fullname

try{
}
catch [$error[0].exception.GetType().fullname]{
}
